### PR TITLE
Check Users service documentation to accordance with code.

### PR DESCRIPTION
**Pull request by VadimDyachenko**
connect #69

### CODE REVIEW

File: architecture/users/readme.md [hunk 1]
```diff
@@ -1,40 +1,34 @@
 # User service
 
-Микросервис предоставляет пользователям (другим микросервисам) данные о пользователях курса (студентах). 
+The service provides the data about course students for users (other services).
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: students ~for~ to users

File: architecture/users/readme.md [hunk 2]
```diff
@@ -1,40 +1,34 @@
...
 
-Данные о пользователях хранятся в CRM, работа с которой осуществляется через API (т.о. на данный момент слой данных - это CRM). Микросервис предоставляет пользователям ограниченный RESTful API (без возможности удаления записей).
+Student's data is stored in the PostgreSQL database. The records in the database are synchronized daily with the source CRM on a schedule.
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: The records in the database are synchronized ~daily~ with the source CRM on ~a~ schedule **once per day**.

File: architecture/users/readme.md [hunk 3]
```diff
@@ -1,40 +1,34 @@
...
+The service provides users with a limited RESTful API without the ability to delete records.
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: The service provides users ~with~ a limited RESTful API with~out the~ ability to read records **only but not to add, update and delete them.**

File: architecture/users/readme.md [hunk 4]
```diff
@@ -1,40 +1,34 @@
...
+* USR-D1 The service receives an incoming request on its URI. Request's parameters are checked for correctness and absence of any unnecessary data (SQL injection, etc.) and a query is performed to the data layer (database)
 
-* USR-D3 Данные для доступа к API CRM хранятся в отдельном файле в виде (а лучше предложить какой-то другой метод хранения):
-    ```
-    x2crm_root=<string>
-    x2crm_user=<string>
-    x2crm_pass=<string>
-    ```
-* USR-D4 Информация по API X2CRM можно найти [тут](http://wiki.x2crm.com/wiki/REST_API_Reference)
+* USR-D2 The service stores limited information about students, necessary for the functioning of other services. All operations are logged in accordance with the established level of logging.
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: The service stores limited information about students (удалить запятую) **that is** necessary for the functioning of other services. All operations are logged ~in accordance with~ **according to** the established level of logging.

File: architecture/users/readme.md [hunk 5]
```diff
@@ -1,40 +1,34 @@
...
 
-* USR-D5 Пользователю должна предоставляться информация об ошибках при выполнении его запроса (достаточная для понимания и исправления им ошибки) - некорректно заданные параметры, отсутствие обязательного параметра, несоответствие формата параметра, отсутствии данных по его запросу и т.д.
+* USR-D3 The user should be provided with information about errors when executing his request, sufficient for understanding and correcting the error (incorrectly set parameters, absence of mandatory parameter, format parameter mismatch, lack of data on its request, etc).
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: ~The user should be provided with information about errors when executing his request, sufficient for understanding and correcting the error (incorrectly set parameters, absence of mandatory parameter, format parameter mismatch, lack of data on its request, etc).~
When an error occurs during user's request user should receive sufficient for understanding and correcting information about this error such as: incorrectly set of parameters, absence of mandatory parameters, format parameter mismatch, lack of data etc.

File: architecture/users/readme.md [hunk 6]
```diff
@@ -1,40 +1,34 @@
...
 
-## Список фичей
+## Features:
 
-### USR-F1 Я как пользователь хочу иметь возможность получить список всех пользователей
+### USR-F1 I, as a user, want to be able to get a list of all students.
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: ~I, as a user, want to be able to get a list of all students.~
**As a user I want to get a list of all students.**

File: architecture/users/readme.md [hunk 7]
```diff
@@ -56,12 +50,12 @@
             "name" : "..."
         },
         ...
-    ]    
+    ]
     ```
 
-### USR-F2 Я как пользователь хочу иметь возможность получить пользователей по их uuid
+### USR-F2 I, as a user, want to be able to get list of students on their uuid
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: ~I, as a user, want to be able~ **As a user I want** to get list of students ~on~ **by** their uuid

File: architecture/users/readme.md [hunk 8]
```diff
@@ -56,12 +50,12 @@
...
 
-* USR-F2-D1 На входе - массив uuid, на выходе - массив users с полями uuid, slack, skype и name.
+* USR-F2-D1 At the input there is an uuid array, at the output there is an array of students with the uuid, slack, skype and name fields.
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: ~At the input there is~ **Input - ** an uuid array, ~at the~ output **-** ~there is~ an array of students with the uuid, slack, skype and name fields.

File: architecture/users/readme.md [hunk 9]
```diff
@@ -90,9 +84,9 @@
     ]
     ```
 
-### USR-F3 Я как пользователь хочу иметь возможность получить пользователей по их slack name.
+### USR-F3 I, as a user, want to be able to get list of students on their slack name.
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: ~I, as a user,~ **As a user I** want ~to be able~ to get list of students ~on~ **by** their slack name.

File: architecture/users/readme.md [hunk 10]
```diff
@@ -90,9 +84,9 @@
...
 
-* USR-F3-D1 Входные данные - массив slackNames, результат - массив users с полями uuid, slack, skype и name.
+* USR-F3-D1 At the input there is a slackNames array, at the output there is an array of students with the uuid, slack, skype and name fields.
```
Comment (OlgaKulikova) [inline architecture/users/readme.md]: ~At the input there is~ **Input -** a slackNames array, ~at the~ output **-** ~there is~ an array of students with the uuid, slack, skype and name fields.

**Comment by VadimDyachenko** [comment]
@OlgaKulikova All mistakes fixed

**Pull request merged**